src/sysinfo.cc

```diff
@@ -224,10 +224,10 @@ std::pair<bool /*validity*/, bool /*value*/> CpuScalingEnabled(int num_cpus) {
   for (int cpu = 0; cpu < num_cpus; ++cpu) {
     std::string governor_file =
         StrCat("/sys/devices/system/cpu/cpu", cpu, "/cpufreq/scaling_governor");
-    if (ReadFromFile(governor_file, &res) && res != "performance") return make_pair(true, true);
+    if (ReadFromFile(governor_file, &res) && res != "performance") return CPUInfo::Scaling::ENABLED;
   }
 #endif
-  return std::make_pair(true, false);
+  return CPUInfo::Scaling::DISABLED;
```

Comment:
should this be disabled or unknown?

Reply:
My thoughts were for DISABLED as for the fallback for line 227 ?
But considering line 217 - could have been fixed also